src/main/java/com/elixr/poc/constants/ApplicationConstants.java

```diff
@@ -9,13 +9,13 @@
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class ApplicationConstants {
 
+    public static final String AMOUNT_MISSING_IN_PURCHASE = "Amount is mandatory";
+
+    public static final String DATE_MISSING_IN_PURCHASE = "Date are mandatory";
     public static final String FIRST_NAME_MISSING_IN_USER = "First name is mandatory";
-    public static final String USER_NAME_MISSING = "Username is mandatory";
```

Comment:
It's better to add validation messages in the message.properties file.

Reply:
Updated